integrations/svelte/server.js

```diff
@@ -0,0 +1,62 @@
+import { Meteor } from 'meteor/meteor';
+import { Translations } from '../imports/api/TranslationsCollection';
+import { i18n } from 'meteor/universe:i18n';
+import { isEqual } from 'lodash-es';
+
+// Import of all files that contain translations, eg:
+import '../imports/locales/en.i18n.yml';
+import '../imports/locales/es.i18n.yml';
+
+// Function for updating i18n translation with translations from database
+function refresh(locale, { translations = {}, updatedAt = new Date() }) {
+  locale = i18n.normalize(locale);
+  i18n._translations[locale] = translations;
+
+  const cache = i18n.getCache(locale);
+  cache.updatedAt = updatedAt.toUTCString();
+  delete cache._json;
+  delete cache._yml;
+  i18n._emitChange(locale);
+}
+
+const mergeTranslations = (baseTranslations, translation) => {
+  return Object.assign({}, translation, baseTranslations);
+};
```

Comment:
This should be a deep merge. Let's change the implementation or at least add a comment regarding that.